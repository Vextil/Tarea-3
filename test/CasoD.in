CrearBinario Uruguay 1 4 Argentina 7 Peru 3 CostaRica 9 Bolivia 11 Paraguay 2 Brasil 8 Ecuador 10 Mexico 12 Chile 5 Colombia 6 Venezuela 4 FIN
ImprimirBinario 4
AlturaCantidadBinario 4
FiltroBinario 4 5 > 1
BuscarABB 4 Uruguay
BuscarABB 1 Uruguay
Encolar 4
Encolar 1
Desencolar
Desencolar
Desencolar
Fin
