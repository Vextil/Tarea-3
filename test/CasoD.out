1>OK
2>
  (4,Venezuela)
 (1,Uruguay)
   (3,Peru)
     (2,Paraguay)
       (12,Mexico)
      (10,Ecuador)
    (9,CostaRica)
        (6,Colombia)
       (5,Chile)
      (8,Brasil)
     (11,Bolivia)
  (7,Argentina)
3>Altura: 8; Cantidad: 12
4>OK
5>
  (4,Venezuela)
 (1,Uruguay)
   (3,Peru)
     (2,Paraguay)
       (12,Mexico)
      (10,Ecuador)
    (9,CostaRica)
        (6,Colombia)
       (5,Chile)
      (8,Brasil)
     (11,Bolivia)
  (7,Argentina)
6>
  (4,Venezuela)
 (1,Uruguay)
  (3,Peru)
   (2,Paraguay)
7>OK
8>OK
9>
  (4,Venezuela)
 (1,Uruguay)
   (3,Peru)
     (2,Paraguay)
       (12,Mexico)
      (10,Ecuador)
    (9,CostaRica)
        (6,Colombia)
       (5,Chile)
      (8,Brasil)
     (11,Bolivia)
  (7,Argentina)

10>
  (4,Venezuela)
 (1,Uruguay)
  (3,Peru)
   (2,Paraguay)

11>La cola estaba vacia.
12>Fin
